lib/mixlib/install/cli.rb

```diff
@@ -75,14 +75,15 @@ def download(product_name)
         if options[:url]
           say artifact.url
         else
+          url = artifact.is_a?(Array) ? artifact.map(&:url).first : artifact.url
```

Comment:
The CLI should not be returning a list of ArtifactInfo instances. What problem are you trying to solve here?